apps/gallery/js/gallery.js

```diff
@@ -400,6 +400,9 @@ function setView(view) {
   }
   switch (view) {
     case LAYOUT_MODE.list:
+      if (!isPhone) {
+        $('thumbnails-list-top').removeAttribute('no-font-fit');
+      }
```

Comment:
Is this header running the font-fit algorithm even though it is never displayed on the phone? 
In fact, none of these gaia header elements are displayed when the app starts up, and I think this is why I never bothered to use this no-font-fit trick for the app.